### PR TITLE
Add some unit tests for tool_calls arguments has null value

### PR DESCRIPTION
**Pull request by weiping-code**
## Change
Add some unit tests for tool_calls arguments has null value
In the example of tool_calls arguments like:
```json
{
  "arg0": "xxx",
  "arg1": null
}
```

### CODE REVIEW

File: langchain4j/src/test/java/dev/langchain4j/service/tool/DefaultToolExecutorTest.java [hunk 1]
```diff
@@ -334,6 +334,56 @@ void should_not_execute_tool_with_null_execution_request() {
...
+    private static class TestNullArgumentTool {
+
+        @Tool
+        public boolean notNull(Integer num) {
+            return num != null;
+        }
+    }
+
+    @Test
+    void should_return_false_when_arg0_is_null() {
```
Comment (dliubarskyi) [inline langchain4j/src/test/java/dev/langchain4j/service/tool/DefaultToolExecutorTest.java]: nit: this could be a `@ParameterizedTest` with `@CsvSource({"{ \"arg0\": null }, false", "{}, false", "{ \"arg0\": 1 }, true"})`

**Review by dliubarskyi — DISMISSED** [review]
@weiping-code thanks a lot!

**Comment by weiping-code** [comment]
@dliubarskyi all done

**Review by dliubarskyi — APPROVED** [review]
@weiping-code thank you!
